src/commands/infra/scaffold.ts

```diff
@@ -301,6 +300,12 @@ export const generateClusterDefinition = (
     Object.keys(fields).forEach(key => {
       fieldDict[key] = fields[key] || "<insert value>";
```

Comment:
Question: Can we always expect `<insert value>`

Reply:
This logic will put `"<insert value>"` as the mapped value in the dict, if the current value is null or an empty string.

Though, if we're requring the same constant string here and a bit below, I'd prefer to see this value be a const. @yradsmikham

Reply:
@mtarng done!